### ISSUE TITLE
changes in how the local database is accessed

### ISSUE THREAD

**Issue by williamgunn** [herrlich10/men2zot#1]
Hey @herrlich10 , not sure if you're still maintaining or using this, but there are some changes to the local database in the latest version of Mendeley which will break your script. We're currently collecting use cases to officially support local database access so we don't break people's scripts when we make changes, so if you'd like to give the Mendeley developers some feedback on what data you need to access so we can make sure to support your use case, please get in touch via email.

**Comment by herrlich10** [comment]
Thanks @williamgunn for the updates about Mendeley development here.

Let's make the situation clearer first. I realized that in the latest Mendeley release (perhaps started from v1.19):
1) There is no more options in the menu to manually backup or restore the database.
2) The newly created sqlite database files seem no longer accessible via even the latest release of sqlite3 command line tools, but complain that "Error: file is encrypted or is not a database". 

Is it because you're really starting to encrypt the sqlite database files, and only accesses via API would be allowed in the future?

Again, many thanks for your information.

**Comment by williamgunn** [comment]
Yes, we would like to get folks on a access mechanism (local access, not only via web) that is more stable than the sqlite. Backup & restore should work, though.

**Comment by EChesters** [comment]
Hi @herrlich10,

Apologies for reaching out on this issue, as I could not find an email to contact you. I am Elizabeth, one of the designers at Mendeley, who would be interested in hearing your feedback as William suggested.

If you are interested, please could you let me know what time would be best for you, at mendeley.design@elsevier.com?

Many thanks,
Elizabeth
